www/less/widenImages.less

```diff
@@ -4,31 +4,9 @@
     display: block;
     margin-left: auto;
     margin-right: auto;
-}
-
-@media (orientation: portrait) {
-    .wideImageOverride {
```

Comment:
I don't think we should remove these... It breaks the image widening javascript logic and makes images take up way too much of the screen on landscape on phones...